### PR TITLE
Fix AKS fetch for deletion

### PR DESCRIPTION
**Pull request by sebader**
Port from https://github.com/Azure/Mission-Critical-Online/pull/539

### CODE REVIEW

File: .ado/pipelines/templates/stages-full-release.yaml [hunk 1]
```diff
@@ -380,6 +380,13 @@ stages:
                     echo "*** Load credentials for AKS Cluster $aksClusterName in $aksClusterResourceGroup"
                     az aks get-credentials --name $aksClusterName --resource-group $aksClusterResourceGroup --overwrite-existing
 
+                    $ingressNamespace = "ingress-nginx"
```
Comment (heoelri) [inline .ado/pipelines/templates/stages-full-release.yaml:383]: Looks good to me. Just one minor thing. We've specified the workload and the chaos namespace in our `configuration.yaml`. Would it make sense to move the ingressNamespace to this file as well?

Comment (sebader) [reply .ado/pipelines/templates/stages-full-release.yaml:383]: you didnt complain about this in the original PR :D but of course you are right! will change

Comment (heoelri) [reply .ado/pipelines/templates/stages-full-release.yaml]: Sorry.

**Review by heoelri — APPROVED** [review]
LGTM!

**Pull request merged**